cpp/ps0000/p0Solution.cpp

```diff
@@ -1,13 +1,17 @@
-#include<iostream> /*"iostream" not "iostream.h" is the correct header file to include for using input-output 
-stream objects in new verions of c++ */
+#include <iostream> // 2/2
+/*"iostream" not "iostream.h" is the correct header file to include 
+for using input-output stream objects in new verions of c++ */
 using namespace std;
 
-int main(void) {
-	/* using void in main function indicates that this functions takes no argument so we can use it as an
-	argument or not */
+int main(void) { // 1/1
+	/* using void in main function indicates that 
+ 	this functions takes no argument */
 
-	cout << "Hello, world!\n"; // for new line we use '\' not '/'  
-	//and for print the output we use "<<" after cout not">>" 
+	cout << "Hello, World!\n"; // 3/4 
+	// for new line we use '\' not '/'  
+	//and for print the output we use "<<" after cout not">>"
 
-	return 0; // returning 0 means successful execution not 1 
-}
\ No newline at end of file
+	return 0; // returning 0 means successful execution not 1
+	// 1/1
+}
+// great work 😎 10/10 ⭐

```

Comment:
- [x] most of code errors and bugs are fixed
- [x] file is in the right path
- [x] +1 marks for your first commit